packages/client/src/store/actions/bookingOperations.ts

```diff
@@ -232,7 +232,7 @@ export const sendICSFile: sendICSFile =
         <a href="${icsFile}">Clicca qui per aggiungere le tue prenotazioni al tuo calendario</a>`;
 
       const handler = CloudFunction.SendEmail;
-      const payload: Omit<SendEmailPayload, "organization"> = {
+      const payload: Omit<SendEmailPayload, "organization" | "emailFrom"> = {
```

Comment:
Can we make the `ClientSendEmailPayload`?
Dispute me on this if the way emails are being sent out is changed in the next PR and this update would immediately become obsolete...

Reply:
I'm kinda confused

Reply:
That type: structure sent out from client to the backend (as cloud function payload) should be named `ClientEmailPayload` and `EmailPayload` should be the structure the `sendEmail` function writes to email delivery document (as `payload`)